code/utils/ps.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+import numpy
+
+import tensorflow as tf
+
+
+def _initialize_pk(shape,boxsize,kmin,dk):
+    """
+       Helper function to initialize various (fixed) values for powerspectra... not differentiable!
+    """
+    I = np.eye(len(shape), dtype='int') * -2 + 1
+
+    W = np.empty(shape, dtype='f4')
+    W[...] = 2.0
+    W[..., 0] = 1.0
+    W[..., -1] = 1.0
+    kedges = np.arange(kmin, np.pi * 10 / 1 + dk/2, dk)
```

Comment:
probably should go to higher k here... not sure if we want to have a kmax or similar option here...